packages/gr/gr.0.0.1/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings to the GR plotting library"
+description: """
+GR is based on an implementation of a Graphical Kernel System (GKS) and OpenGL. As a self-contained system it can quickly and easily be integrated into existing applications.
+GR is characterized by its high interoperability and can be used with modern web technologies and mobile devices."""
+maintainer: ["Marcello Seri"]
+authors: ["Marcello Seri"]
+license: "ISC"
+homepage: "https://github.com/mseri/ocaml-gr"
+bug-reports: "https://github.com/mseri/ocaml-gr/issues"
+depends: [
+  "ocaml" {>= "4.06"}
+  "dune" {>= "1.11"}
+  "ctypes" {>= "0.15"}
+  "ctypes-foreign"
+  "base-unix" {with-test}
+  "owl" {with-test & >= "0.6.0"}
+  "stdcompat" {>= "9"}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
    "@runtest" {with-test & ocaml:version >- "4.07"}
```
I'll fix the tests upstream, there is no point in making a new release just for that